shop/src/main/java/ro/msg/learning/shop/config/StrategyConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package ro.msg.learning.shop.config;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import ro.msg.learning.shop.repository.LocationRepository;
+import ro.msg.learning.shop.repository.StockRepository;
+
+@Configuration
+@AllArgsConstructor
+public class StrategyConfiguration {
+
+    //private final StockRepository stockRepository;
```

Comment:
Do not merge commented cod in master. If this class is not ready, exclude it from the Pull Request.

Reply:
Changed